setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='django-proctor',
-    version='1.0.6',
+    version='1.1.0',
```

Comment:
Oh, could you also add the following below `Framework :: Django :: 1.11` in the `classifiers` list:
```py
        'Framework :: Django :: 2.0',
        'Framework :: Django :: 2.1',
        'Framework :: Django :: 2.2',
```